### PR TITLE
Revise tx set

### PR DESCRIPTION
**Pull request by ycho**

### CODE REVIEW

File: src/context.rs [hunk 1]
```diff
@@ -557,38 +552,24 @@ pub fn get_tx_set(
...
-  }
-
-  if tx_size_sqr == TxSize::TX_16X16 {
-    TxSet::TX_SET_DTT4_IDTX
+      TxSet::TX_SET_INTER_1
+    }
+  } else if tx_size_sqr_up == TxSize::TX_32X32 {
+    TxSet::TX_SET_DCTONLY
+  } else if use_reduced_set || tx_size_sqr == TxSize::TX_16X16 {
+    TxSet::TX_SET_INTRA_2
```
Comment (ycho) [inline src/context.rs]: @lu_zero, We've had one missing case like here.

File: src/transform/mod.rs [hunk 2]
```diff
@@ -238,23 +238,15 @@ pub enum TxSet {
...
   TX_SET_DCTONLY,
   // DCT + Identity only
-  TX_SET_DCT_IDTX,
+  TX_SET_INTER_3, // TX_SET_DCT_IDTX
   // Discrete Trig transforms w/o flip (4) + Identity (1)
-  TX_SET_DTT4_IDTX,
+  TX_SET_INTRA_2, // TX_SET_DTT4_IDTX
   // Discrete Trig transforms w/o flip (4) + Identity (1) + 1D Hor/vert DCT (2)
-  // for 16x16 only
-  TX_SET_DTT4_IDTX_1DDCT_16X16,
```
Comment (ycho) [inline src/transform/mod.rs]: @lu_zero, And three entries of TxSet enum was obsolete, i.e. discarded when av1 was finalized.

**Comment by ycho** [comment]
> Could you please add in the commit a bit more information on why this change and what's its impact on awcy?

Sure, I've already added two as inline comments.
1) There has been three obsolete entries in TxSet eum
2) There was one missing case in get_tx_set() function
3) I thought it is helpful to write the function similar to av1 spec for easier maintenance, unless it slows down rav1e.

**Comment by ycho** [comment]
I've also run awcy just now for several levels, 0,2,6,10. I hope there is no change there.

**Comment by ycho** [comment]
No change with awcy runs at [speed 2](https://beta.arewecompressedyet.com/?job=master_caef43f3f_s2%402020-01-18T18%3A02%3A22.729Z&job=test_revise_s2%402020-01-18T17%3A49%3A46.108Z),
[speed 6](https://beta.arewecompressedyet.com/?job=master_caef43f3f_s6%402020-01-18T18%3A02%3A40.078Z&job=test_revise_s6%402020-01-18T17%3A49%3A24.530Z).
and [speed 0](https://beta.arewecompressedyet.com/?job=master_caef43f3f_s0%402020-01-18T18%3A03%3A02.154Z&job=test_revise_s0%402020-01-18T17%3A48%3A52.953Z) is still running....

**Comment by ycho** [comment]
Reviewers, this PR is required to land for enabling inter mode tx partition with one level split.

File: src/transform/mod.rs (hunk 2 above)
Comment (lu-zero) [reply src/transform/mod.rs]: Should be written in the commit message :)

**Comment by ycho** [comment]
> It is fine, just another commit nit if you want.

Fixed and also stated in commit message what obsolete tx_sets are removed.

**Pull request merged**